libraries/AP_Scheduler/AP_Scheduler.tpp

```diff
@@ -0,0 +1,186 @@
+extern const AP_HAL::HAL& hal;
```

Comment:
Dear @dgrat , please, not say "could", you need to do it, because it's declared on GPLv3 statement. All people know it trackable, then, for what Richard Stallman created GNU Licence, to add it at first time and later delete it and to be it trackeable on github? No @dgrat , it's protect the future changes, your mind and ideas was opened from this code, like me, so, you have respect that.

Author comments will be in if the author wants, if you are coming and working for an enterprise to they say that you don't have to write your author comments, that is your matter.

@lucasdemarchi are writing all the new structure APM HAL for linux under Intel statement and i don't know why, he must have his reasons.

This is a example from the Linus Torvalds kernel on github and how do you need to do an author comment https://github.com/hiro2233/linux/blob/master/drivers/i2c/i2c-core.c

Greetings,
Hiroshi.